www/source/docs/run-packages-update-strategy.html.md

```diff
@@ -24,7 +24,7 @@ Once the update leader finds a new version it will update and wait until all oth
 
 If your service group is also running with the `--topology leader` flag, the leader of that election will never become the update leader, so all followers within a leader topology will update first.
 
-It's important to note that because we must perform a leader election to determine an update leader, *you need to have at least 3 supervisors running a service group to take advantage of the rolling update strategy*.
+It's important to note that because we must perform a leader election to determine an update leader, *you need to have at least 3 supervisors running in a service group that is in the same application and environment group (if you have specified those tags in your ring) to take advantage of the rolling update strategy*.
```

Comment:
I think it would be good to link to docs that define `application and environment group`. I probably have not been paying close enough attention, but I'm not sure what those are.

Reply:
Those don't actually exist yet -_-;;; 

I will take an action to add some